src/components/Button.tsx

```diff
@@ -1,26 +1,25 @@
 import React, { useState } from "react";
 import { Pressable, StyleSheet, View, Text } from "react-native";
+import type { StyleProp } from "react-native";
 import Icon from "react-native-vector-icons/FontAwesome";
 
 import { PRIMARY_COLOR, PRIMARY_LIGHT, TEXT_COLOR } from "../util/colors";
 
 export interface ButtonProps {
 	text?: string;
-	image?: string;
+	iconName?: string;
 	accessibilityLabel: string; // not optional for this project.
 	onPress: VoidFunction;
 	//eslint-disable-next-line @typescript-eslint/ban-types
-	style?: object; // may be unsafe, but this is the type provided
-	// by Stylesheet documentation:
-	// https://reactnative.dev/docs/stylesheet#compose
+	style?: StyleProp<object>; // TODO: update generic from "object"
 }
 
 /**
  * Peer-styled button element.
  */
 export const Button: React.FC<ButtonProps> = ({
 	text,
-	image,
+	iconName: image,
```

Comment:
This can be fixed later but at this point i would just get rid of "image" entirely